itest/test_list_on_test.go

```diff
@@ -23,6 +23,18 @@ var testCases = []*testCase{
 		name: "multi address",
 		test: testMultiAddress,
 	},
+	{
+		name: "re-org mint",
+		test: testReOrgMint,
+	},
+	{
+		name: "re-org send",
+		test: testReOrgSend,
+	},
+	{
+		name: "re-org mint and send",
+		test: testReOrgMintAndSend,
+	},
```

Comment:
We should probably add a comment to explain that they should be run first because otherwise the postgres itests are flaky.

Reply:
Good point, added a comment.